app/models/fruit.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Fruit < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Right now, if you try to create an fruit without one of the required fields, it fails, but does not tell the user that it failed.  That could be an interesting next step for the project.